eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Suggestion/Formatter/YamlSuggestionFormatter.php

```diff
@@ -19,7 +19,7 @@ public function format(ConfigSuggestion $configSuggestion)
         $suggestion = $configSuggestion->getSuggestion();
         if ($suggestion) {
             $yamlConfig = Yaml::dump($suggestion, 8);
-            if (php_sapi_name() !== 'cli') {
+            if (PHP_SAPI !== 'cli') {
```

Comment:
Rule `function_to_constant`. I think also valuable for performance (microseconds, but still...!). Also it's the only one occurrence AFAIR